### PR TITLE
added rsvp list

### PR DESCRIPTION
**Pull request by RachellCalhoun**
I forgot to make a newbranch. This adds a list of users that rsvped. Have to make it pretty. not sure what issue this was?

### CODE REVIEW

**Comment by jinpark** [comment]
For #3 

Awaiting pretty version!

File: rsvp/templates/rsvp/event_detail.html [hunk 1]
```diff
@@ -26,6 +26,9 @@
       {% elif rsvp and rsvp.first.attending %}
         <button class="rsvp-submit right mdl-button mdl-js-button mdl-button--raised" data-event-id="{{ event.id }}">Un-rsvp<img class='loading hidden' src="{% static 'rsvp/loading.gif' %}" alt=""></button>
       {% endif %}
+      {%for rsvp in rsvplist%}
```
Comment (hassanabidpk) [inline rsvp/templates/rsvp/event_detail.html]: @RachellCalhoun  Can you use Table from here http://www.getmdl.io/components/index.html#tables-section  May be include S.No , UpdatedAt and username

**Comment by jinpark** [comment]
image
can we push the RSVP list down a bit so its below the (first meeting) title?

**Comment by jinpark** [comment]
looks good, can you clear out the debug.log file and then lets get this merged!

**Comment by hassanabidpk** [comment]
:+1:

**Pull request merged**